homeassistant/components/binary_sensor/nest.py

```diff
@@ -60,11 +61,8 @@
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Setup Nest binary sensors."""
-    if discovery_info is None:
```

Comment:
The configurators shows up and works as intended. But I see now my code introduces two problems. Both sensors load with a error at startup if Nest is not configured. And a restart is needed to load the sensors after configuration is complete.